lib/rest/RequestHandler.js

```diff
@@ -236,7 +236,7 @@ class RequestHandler {
                         }
 
                         if(resp.statusCode !== 429) {
-                            this._client.emit("debug", `${body && body.content} ${now} ${route} ${resp.statusCode}: ${latency}ms (${this.latencyRef.latency}ms avg) | ${this.ratelimits[route].remaining}/${this.ratelimits[route].limit} left | Reset ${this.ratelimits[route].reset} (${this.ratelimits[route].reset - now}ms left)`);
+                            this._client.emit("debug", `${body && body.content || "None"} ${now} ${method} ${route} ${resp.statusCode}: ${latency}ms (${this.latencyRef.latency}ms avg) | ${this.ratelimits[route].remaining}/${this.ratelimits[route].limit} left | Reset ${this.ratelimits[route].reset} (${this.ratelimits[route].reset - now}ms left)`);
```

Comment:
None is too common and isn't explicit to being what exactly is None. Also sometimes errors actually respond with None so its safer to use a different term imo. `undefined` is fine if it is prefixed with what is undefined. In this case just adding Body Content is enough imo.
```suggestion
                            this._client.emit("debug", `Body Content: ${body && body.content} ${now} ${method} ${route} ${resp.statusCode}: ${latency}ms (${this.latencyRef.latency}ms avg) | ${this.ratelimits[route].remaining}/${this.ratelimits[route].limit} left | Reset ${this.ratelimits[route].reset} (${this.ratelimits[route].reset - now}ms left)`);
```